openapi.yaml

```diff
@@ -1,190 +1,513 @@
+    description: Unique Park ID
+    required: true
+    type: string
+  parkPostBody:
+    name: parkBody
+    in: body
+    description: A park body object
+    required: true
+    schema:
+      $ref: '#/definitions/PostParkModel'
+  parkPatchBody:
+    name: parkBody
+    in: body
+    description: A park body object
+    required: true
+    schema:
+      $ref: '#/definitions/PatchParkModel'
+  ownerPostBody:
+    name: ownerPostBody
+    in: body
+    description: An owner body object
+    required: true
+    schema:
+      $ref: '#/definitions/PostOwnerModel'
+  ownerPatchBody:
+    name: ownerPatchBody
+    in: body
+    description: An owner body object
+    required: true
+    schema:
+      $ref: '#/definitions/PatchOwnerModel'
 definitions:
-  Meta:
-    properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPages:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
   SelfLink:
     properties:
       self:
         type: string
+        example: request_url_here
         format: url
         description: Self-link of current resource
-  PaginationLinks:
+  OwnerResource:
+    required: [id, type, attributes]
     properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
+      id:
         type: string
-        format: url
-        description: The last page of data
-      prev:
+        example: '0001'
+        description: A unique ID of a park owner
+      type:
         type: string
-        format: url
-        description: The previous page of data
-      next:
+        description: type of object, should be owner
+        example: owner
+      attributes: &ownerAttributes
+        required: [ownerName]
+        properties:
+          ownerName:
+            type: string
+            description: Name of park owner / organization
+            example: City of Corvallis
+  PostOwnerModel:
+    required: [type, attributes]
+    properties:
+      type:
         type: string
-        format: url
-        description: The next page of data
-  PetResource:
+        description: type of object, should be owner
+        example: owner
+      attributes: *ownerAttributes
+  PatchOwnerModel:
+    required: [id, type]
     properties:
       id:
         type: string
-        description: A unique ID of pet
+        example: '0001'
+        description: A unique ID of a park owner
       type:
         type: string
-        enum: [pet]
-      links:
-        $ref: '#/definitions/SelfLink'
+        description: type of object, should be owner
+        example: owner
       attributes:
+        required: [ownerName]
```

Comment:
I don't think `ownerName` should be required here since PATCH requests usually make all attribute fields optional